lib/benchee/configuration.ex

```diff
@@ -13,30 +13,31 @@ defmodule Benchee.Configuration do
   }
 
   defstruct [
-    parallel:          1,
-    time:              5,
-    warmup:            2,
-    formatters:        [Console],
+    parallel:             1,
+    time:                 5,
+    warmup:               2,
+    formatters:           [Console],
     print: %{
-      benchmarking:    true,
-      configuration:   true,
-      fast_warning:    true
+      benchmarking:       true,
+      configuration:      true,
+      fast_warning:       true
     },
-    inputs:            nil,
+    inputs:               nil,
     # formatters should end up here but known once are still picked up at
     # the top level for now
     formatter_options: %{
       console: %{
-        comparison:    true
+        comparison:       true,
+        extended_options: false
```

Comment:
I think I'd call it `extended_statistics` and my initial urge would be to enable it by default. Not sure, but this way it'd be more discoverable. I also tend to flood people with things though :D